nemo/collections/asr/models/hybrid_rnnt_ctc_models.py

```diff
@@ -229,7 +235,10 @@ def transcribe(
                 self.joint.unfreeze()
                 if hasattr(self, 'ctc_decoder'):
                     self.ctc_decoder.unfreeze()
-        return hypotheses, all_hypotheses
+        if logprobs:
+            return logits_list
```

Comment:
Can you do logprobs, None so signature is always 2 elements out

Reply:
yes I could, but I think it is better to return one, because it replicates EncDecCTCModel.transcribe(files, logprobs=True) function which present in many tutorials, for instance   https://github.com/NVIDIA/NeMo/blob/88816f0b87364ee539731a0f59f298e958ab68f9/tutorials/asr/Offline_ASR.ipynb#L248

Reply:
Oh I see, EncDecCTCModel.transcribe() always return one

Reply:
done